python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -40,6 +48,7 @@ _STRING_AGGS = {
     "nth",
     "collect",
     "unique",
+    "cumcount",
```

Comment:
Not the right place, but can we run `cumcount` on decimal columns as well?

Reply:
Yes. 

https://github.com/rapidsai/cudf/pull/7759#discussion_r605407868
> pandas gives `cumcount` for entire groupby object. Not on individual columns.

Reply:
I think we need to add `cumcount` to the `_DECIMAL_AGGS` dict then as well